[DEV-4.5] Game Music

imranmuyingo opened the pull request:
I have implemented the game music by playing background music whilst the user is playing the game. The user can click on button "stop" and "play" to control the music

imranmuyingo committed:
4.5-game_music

I have implemented the game music by playing background music whilst the user is playing the game. The user can click on button "stop" and "play" to control the music

gideonmehna commented on views/BoggleView.java:
```diff
@@ -110,12 +117,18 @@ public void initUI() {
+        var scene = new Scene(borderPane, 1200, 600);
         this.stage.setScene(scene);
         this.stage.show();
+        Media media = new Media(Paths.get(MEDIA_URL).toUri().toString());
```
This is great! I wonder if we should care for Dependency Inversion

gideonmehna reviewed (commented):
I was just wondering if we need to care for dependency inversion

imranmuyingo committed:
feature/4.5-game_music

This feature allows background music to be played whilst playing the game. The user can use buttons "stop" and "play" to control the music

imranmuyingo committed:
Update workspace.xml

imranmuyingo committed:
Merge branch 'develop' into feature/4.5-game_music
